### PR TITLE
[MNOE-872] Expose session timeout config

### PR DESCRIPTION
**Pull request by enizor**
Allows the frontend to warn the user when he will get logged off due to inactivity

### CODE REVIEW

**Comment by enizor** [comment]
@ouranos for the review

File: api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb [hunk 1]
```diff
@@ -25,5 +25,5 @@ angular.module('mnoEnterprise.configuration', [])
   .constant('APP_NAME', <%= MnoEnterprise.app_name.to_json %>)
   .constant('URL_CONFIG', <%= Hash(Settings.url_config).to_json %>)
   .constant('DEVISE_CONFIG', <%=
-      Hash(Settings.devise).merge(min_password_length: Devise.password_length.min).to_json
+      Hash(Settings.devise).merge(min_password_length: Devise.password_length.min, timeout_in: (Devise.timeout_in || 0).seconds).to_json
```
Comment (ouranos) [inline api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb]:
```ruby
2.3.3 :006 > 5.minutes.to_i
 => 300 
2.3.3 :007 > nil.to_i
 => 0
```

Comment (ouranos) [reply api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb]: Actually you could leave: `timeout_in: Devise.timeout_in` which will be translated to the number of second or to `null` when passed to `to_json`. You can then handle the default  in the frontend (see my review)

**Comment by MAhsenArif** [comment]
This can be closed as task is continued on https://github.com/maestrano/mno-enterprise/pull/654

**Pull request closed without merging**